components/footer.py

```diff
@@ -45,64 +49,73 @@ def my_footer():
                                     c="white",
                                 ),
                                 href=TextFooter.report_issues_link.value,
-                            )
+                            ),
                         ),
-                        span={"base": 6, "md": 1},
+                        span={"base": 12, "md": 1},
                     ),
                     dmc.GridCol(
-                        dmc.Center(
-                            dmc.Anchor(
-                                dmc.Image(
-                                    src="/assets/media/github-white-transparent.png",
-                                    maw=45,
-                                    alt="github logo",
+                        dmc.Group(
```

Comment:
same comment as before, why do we need this group? It works also without this group

Reply:
I treat two elements in one line as a group to center them for mobile platform